src/doctor.ts

```diff
@@ -39,6 +48,10 @@ export class DoctorProvider implements Disposable {
         ViewColumn.Active,
         { enableCommandUris: true }
       );
+      if (this.extensionContext)
+        this.doctor.iconPath = Uri.file(
+          path.join(this.extensionContext.extensionPath, "icons", "doctor.svg")
```

Comment:
I was thinking of having different doctors depending on the theme, but that seemed to be too much on the nose :sweat_smile: